From a1270429a0b81dadb64f7914a6fba68bc090b515 Mon Sep 17 00:00:00 2001
From: Markus Armbruster <armbru@redhat.com>
Date: Thu, 28 Jul 2016 06:30:09 +0200
Subject: [PATCH 24/35] rhel: Revert unwanted inconsequential changes to
 ivshmem

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1469687410-14432-2-git-send-email-armbru@redhat.com>
Patchwork-id: 71504
O-Subject: [RHEV-7.3 qemu-kvm-rhev PATCH v2 1/2] rhel: Revert unwanted inconsequential changes to ivshmem
Bugzilla: 1333318
RH-Acked-by: Auger Eric <eric.auger@redhat.com>
RH-Acked-by: Andrew Jones <drjones@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

This reverts the ivhsmem part of commit ea18abc "Enable/disable
devices for RHEL 7".  It didn't actually enable or disable anything.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/misc/ivshmem.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/hw/misc/ivshmem.c b/hw/misc/ivshmem.c
index ecb5f5a..e40f23b 100644
--- a/hw/misc/ivshmem.c
+++ b/hw/misc/ivshmem.c
@@ -142,7 +142,6 @@ static inline bool ivshmem_is_master(IVShmemState *s)
     return s->master == ON_OFF_AUTO_ON;
 }
 
-
 static void ivshmem_update_irq(IVShmemState *s)
 {
     PCIDevice *d = PCI_DEVICE(s);
@@ -818,7 +817,6 @@ static void ivshmem_disable_irqfd(IVShmemState *s)
     }
 
     msix_unset_vector_notifiers(pdev);
-    return;
 }
 
 static void ivshmem_write_config(PCIDevice *pdev, uint32_t address,
-- 
1.8.3.1

